pom.xml

```diff
@@ -47,14 +75,19 @@
       <dependency>
         <groupId>org.testcontainers</groupId>
         <artifactId>testcontainers-bom</artifactId>
-        <version>1.19.0</version>
+        <version>${testcontainers-bom.version}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
     </dependencies>
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>${jaxb-api.version}</version>
+    </dependency>
```

Comment:
Added to fix version mismatch:
![image](https://github.com/user-attachments/assets/2262bb95-7980-4320-8574-5c8007ce4c69)

Other Folio modules also use it:
https://github.com/search?q=org%3Afolio-org%20jaxb-api&type=code